apps/web/src/app/register/page.tsx

```diff
@@ -20,11 +20,11 @@ export default function RegisterPage() {
         <Typography variant="p">
           Verzameling regels voor &quot;Individuele inkomenstoeslag&quot; - Gemeente Utrecht Betreft het object:
           &quot;Aanvraag&quot; Drie regelgroepen met in ALEF uitgewerkte HTML pagina&apos;s. Bronnen:{' '}
-          <Link href="https://pki.utrecht.nl/Loketproduct/0c17f7cd409dc999eb351883a138ca3d">
-            https://pki.utrecht.nl/Loketproduct/0c17f7cd409dc999eb351883a138ca3d
+          <Link href="https://loket.digitaal.utrecht.nl/nl/form/individuele-inkomenstoeslag-aanvragen/startpagina">
```

Comment:
Wat ik niet begrijp is dat ik nog steeds op localhost de oude links op de voorbeeld pagina heb staan. Dus https://pki.utrecht.nl/Loket/product/0c17f7cd409dc999eb351883a138ca3d terwijl ik deze change heb gedaan